epictrack-web/src/components/workPlan/WorkPlanContext.tsx

```diff
@@ -22,6 +23,7 @@ interface WorkplanContextProps {
   setWork: Dispatch<SetStateAction<Work | undefined>>;
   firstNations: WorkFirstNation[];
   setFirstNations: Dispatch<SetStateAction<WorkFirstNation[]>>;
+  status: Status[];

```

Comment:
will the status data be needed across all tabs. Or only needed in the status tab. If only in the status tab I suggest we create a separate context for the status tab. StatusContext

Reply:
i am not sure if we will be needing it accross other tabs but we need it on the WorkPlanContainer.tsx for the tab options